transport/transport.go

```diff
@@ -20,17 +20,18 @@ func Dial(addr string) (net.Conn, error) {
 }
 
 //Forward forward remote and local connection
-func Forward(dst io.Writer, src io.Reader) error {
+func Forward(dst io.Writer, src io.Reader) (int64, error) {
```

Comment:
add return value comments

Reply:
// It returns the number of bytes
// write to dst and the first error encountered while writing, if any.